style.css

```diff
@@ -419,3 +450,183 @@ form {
 #indicator_last {
   padding-bottom: 8px;
 }
+
+@media (min-width: 992px) {
+  .nav-bar-list {
+    display: flex;
+    justify-content: center;
+  }
+
+  .nav-list {
+    font-family: "Inter", serif;
+    font-weight: 600;
+    text-decoration: none;
+    color: #172b4d;
+    padding: 20px;
+  }
+
+  .menu-bar {
+    display: none;
+  }
+
+  body {
+    background-image: url(images/background-desktop.png);
+    background-repeat: no-repeat;
+    background-position-y: -11px;
+    background-position-x: 19px;
+    background-size: 100%;
+  }
+
+  .container {
+    background-image: none;
+  }
+
+  .text {
+    font-size: 15px;
+    width: 70%;
+    padding: 4px 24px;
+    margin-top: -25px;
+    line-height: 32px;
+  }
+
+  .icons {
+    margin-bottom: 50px;
+  }
+
+  .item-1 {
+    margin-top: 250px;
+    margin-left: 335px;
+  }
+
+  .heading-container {
+    margin-top: 32px;
+  }
+
+  .work {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .second-class {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    padding: 0 59px;
+    margin-top: -75px;
+  }
+
+  .container-6 {
+    width: 270px;
+    height: 285px;
+    padding-bottom: 42px;
+  }
+
+  .image-1 {
+    margin-top: -3px;
+  }
+
+  .first-class {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+
+  .container-3 {
+    width: 282px;
+    height: 381px;
+    margin: 10px;
+  }
+
+  .button-2h {
+    height: 39px;
+    width: 97px;
+    font-size: 15px;
+  }
+
+  .button-1 {
+    padding: 0 7px;
+  }
+
+  .paragraph-2 {
+    margin: 39px 208px;
+  }
+
+  #container-5 {
+    background-repeat: no-repeat, no-repeat;
+    background-position: top right, left;
+    background-image: url("/images/Frame.svg"), url("/images/frame.png");
+    margin: 30px 0;
+    background-size: 144px;
+  }
+
+  .button-3h {
+    width: 128px;
+    height: 44px;
+    font-size: 15px;
+    margin-bottom: 77px;
+  }
```

Comment:
- [ ] [OPTIONAL] This is compulsory you implement it but not enough for me not to approve because you have a great design here. Kindly ensure that the `Get My REsume` button has equal padding within it.
#### Your design:
![Screenshot from 2022-02-17 22-49-35](https://user-images.githubusercontent.com/58692590/154577284-4af6cde9-ea81-423a-b131-66487b19830e.png)

#### Template Provided:
![Screenshot from 2022-02-17 22-49-15](https://user-images.githubusercontent.com/58692590/154577378-195ab147-50fd-4422-96f8-43b9065e4b5a.png)